### ISSUE TITLE
Remove invalid project 'path'

### ISSUE THREAD

**Issue by jaraco** [pypa/warehouse#4123]
In [sourceforge 633](https://sourceforge.net/p/pypi/support-requests/633/), I made a claim to the `path` name in PyPI.

This request is a similar one, but based on PEP-541. The project is invalid as it is a squatting package with no uploads.

**Comment by yeraydiazdiaz** [comment]
Hi @jaraco, apologies for the delay.

I tried contacting the user that uploaded the package but the email was no longer active. 

Investigating further I found a [GitHub repo with the same name](https://github.com/mruffalo/path) and contacted its maintainer. He replied promptly saying that he had no relation with the original author and the repo is just a copy of the sourceforge contents. He also stated that he did not have any interest in the "path" PyPI package so we will go ahead and transfer the project to you.
